config/zones.json

```diff
@@ -1,4 +1,11 @@
 {
+  "AU": {},
+  "BR": {},
+  "CA": {},
+  "CL": {},
+  "JP": {},
+  "MY": {},
+  "US": {},
```

Comment:
Would be cool if we could get the generate-bounding-boxes script generate bounding boxes here as well

Reply:
TODO :)